### ISSUE TITLE
Story page social minor issues

### ISSUE THREAD

**Issue by tskillern** [CUCentralAdvancement/essential-cu#81]
cc @pspaeth 

@kmackvolt From the dummy content, we were able to review and merge your work. Thanks. Once content via CMS is rendered on the FE, we'll see more, but so far so good. 

Here are issues we did notice, all social. I consider No. 1 below to be a priority _to at least investigate_ but don't want to spend too much time on it if we can't figure out why @duquennoisa is the only instance of this bug. No. 2 and No. 3 are not priorities. If easy fix later on, OK.

1. @duquennoisa is not seeing social icons in Firefox but she does see them in Chrome. I see them in Firefox. 

![no social icons on firefox](https://user-images.githubusercontent.com/70039323/94730910-33a22300-0321-11eb-9271-fe0cad9bf5c7.png)

2. In LinkedIn, the image is not coming through.

![linkedin no image](https://user-images.githubusercontent.com/70039323/94730982-4f0d2e00-0321-11eb-9915-e94e24dca511.png)
  
3. Twitter is pulling only the URL -- no other info like title or subtitle or image. Several weeks ago, @kreynen and I both noticed this issue a while back. My Twitter accounts seem only to pull through the URL and nothing else, but that may not be the same for all accounts. 

![twitter url only](https://user-images.githubusercontent.com/70039323/94731112-7b28af00-0321-11eb-857d-f15240708fd2.png)

When I attempt to share a story from colorado.edu for example, I do get more info supplied: 

![Screen Shot 2020-09-30 at 1 34 51 PM](https://user-images.githubusercontent.com/70039323/94731333-d3f84780-0321-11eb-8fe1-912723570345.png)

**Comment by kmackvolt** [comment]
@tskillern 
1. Social share not showing up in Anne's Firefox - I have tested on both Mac and Window FF browser many times and can not duplicate the problem. Could have been a cache issue or JS error or other unrelated issue? With the adblocker I am using, it did not affect sharing.

2-3. I added more OG and Twitter-specific tags, and hard-coded a locally-served image instead of relying on the placeholder API which I suspect may have been causing some issues.

Note that once an image is used in a Twitter card post, it can NOT be used again and won't show up. That would explain why images weren't showing up with Twitter with the single placeholder API call. Even with the hard-coded local image I have in there now, that image would only show up the first time someone shares a Tweet with it (which I have, as pictured).

This is a good utility for validating a Twitter share:
https://cards-dev.twitter.com/validator

**In short - at this point, I believe that all sharing functionality should be working ok once we have actual images with unique URLs coming through and rendered.**

Attached are screenshots of actual posts I just made. 

![stage FB cap](https://user-images.githubusercontent.com/40502718/94849174-dd44eb00-03d9-11eb-8363-7a3eca45c679.JPG)
<img width="437" alt="stage LinkedIn cap" src="https://user-images.githubusercontent.com/40502718/94849175-dddd8180-03d9-11eb-979e-78d184b92a05.PNG">
<img width="454" alt="stage Twitter cap" src="https://user-images.githubusercontent.com/40502718/94849176-de761800-03d9-11eb-8cbd-60ba2955fa27.PNG">

**Comment by tskillern** [comment]
OK, thanks @kmackvolt This works for me then. I trust that it's working then. I don't think we should spend more time on it. I couldn't reproduce her error either.
